bosh-release/jobs/wso2is/templates/config/registry.xml

```diff
@@ -1,182 +0,0 @@
-<?xml version="1.0" encoding="ISO-8859-1"?>
```

Comment:
What would be the reason for removing this file?

Reply:
@imesh I have actually performed a `git mv` to the total `bosh-release` folder from the project root to the new `pattern-1` directory.

Reply:
The PR shows some files have been moved but some as deleted and added as new.